float32    TransformLength
bool[4]     JointMode
float32[4]  JointData